src/Command/MeilisearchClearCommand.php

```diff
@@ -4,27 +4,29 @@
 
 namespace Meilisearch\Bundle\Command;
 
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 final class MeilisearchClearCommand extends IndexCommand
 {
+    protected static $defaultName = 'meilisearch:clear|meili:clear';
+    protected static $defaultDescription = 'Clear the index documents';
+
     public static function getDefaultName(): string
     {
-        return 'meilisearch:clear|meili:clear';
+        return self::$defaultName;
     }
 
     public static function getDefaultDescription(): string
     {
-        return 'Clear the index documents';
+        return self::$defaultDescription;
     }
```

Comment:
Revert this, as there is no point to do this at current time. When we'll be able to provide info via attributes we will.
Same in other commands